content/docs/05.concepts/expression/04.function.md

```diff
@@ -96,17 +127,28 @@ The `fromJson` function will convert any JSON string to an object allowing to ac
 If you were using Kestra in a version prior to [v0.18.0](../../../blogs/2024-08-06-release-0-18.md), this function used to be named `json()`. We've renamed it to `fromJson` for more clarity. The renaming has been implemented in a non-breaking way — using `json()` will raise a warning in the UI but it will still work.
 ::
 
+## kv
 
-## yaml
+The `kv` function will retrieve the value for the given key from the KV store.
 
-The `yaml` function will convert any string to an object allowing to access its properties.
+```twig
+{{ kv('my_key') }}
+```
+
+By default, it will search for the key within the same namespace to which the flow belongs. And in case, the key is not found in the namespace, it will return the value as `null`.
```

Comment:
```suggestion
By default, it will search for the key within the same namespace as the flow. If the key is not found in the namespace, it will return the value as `null`.
```